api/incidents/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class IncidentsConfig(AppConfig):
+    name = 'incidents'
```

Comment:
Ensure you have a blank line after the end of every file.

Reply:
Thanks for pointing out. I have made the changes.